core/docker-entrypoint.sh

```diff
@@ -4,12 +4,62 @@ export QDB_PACKAGE=${QDB_PACKAGE:-docker}
 
 QUESTDB_DATA_DIR=${QUESTDB_DATA_DIR:-"/var/lib/questdb"}
 IGNORE_DATA_ROOT_MOUNT_CHECK=${IGNORE_DATA_ROOT_MOUNT_CHECK:-"false"}
-IGNORE_FIND_AND_OWN_DIR=${IGNORE_FIND_AND_OWN_DIR:-"false"}
+FIND_AND_OWN_DIR=${FIND_AND_OWN_DIR:-"false"}
 RUN_AS_ROOT=${RUN_AS_ROOT:-"false"}
+DEFAULT_LOCAL_DIRS=${DEFAULT_LOCAL_DIRS:-"/conf /public /db"}
+
+cat << EOF > /tmp/help_found_files_not_owned
+
+!!! The following files/directories have been found not to be owned by the questdb user/group, which may cause QuestDB to fail to start. !!!
+
+To allow the questdb container image to change the file ownership for you, please run the container with the -e FIND_AND_OWN_DIR=true option.
+This will only work if you don't use --user argument when running the container image
+For example, you can use the command:
+  docker run -it -v /local-questdb-path:/var/lib/questdb -e FIND_AND_OWN_DIR=true questdb/questdb
+
+
+Alternatively, you can change the file ownership yourself.
+QuestDB usually runs under the questdb user with user id 10001 and group id 10001.
+To change the file ownership to questdb user, you can run the command chown 10001:10001 /local-questdb-path -R
+
+
+If you are running QuestDB on your local machine under a normal user (e.g., user id 1000) and you don't want to change the file permissions, 
+you can run the questdb container image with the --user $(id -u):$(id -g) option. 
+For example, you can use the command:
+  docker run -it -v /local-questdb-path:/var/lib/questdb --user $(id -u):$(id -g) questdb/questdb
+
+
+If you want to run QuestDB as root, you can add the RUN_AS_ROOT environment variable when running questdb. 
+For example, you can use the command:
+  docker run -it -v /local-questdb-path:/var/lib/questdb -e RUN_AS_ROOT=true questdb/questdb
+
+EOF
+
+
+if [ -z "${LOCALDIRS:-}" ]
```

Comment:
what is `LOCALDIRS` ? Can we add a comment why we need this and what is the intent of ths branch?